README.md

```diff
@@ -31,11 +31,13 @@ Table of Contents
 
 ## Get Involved
 
-There is a lot to do! If you are interested in getting involved, please join
-the mailing lists and attend the community meetings. If you're interested in
-contributing to a specific part of the project, please join the appropriate
-special interest group (SIG). Details for all of these items are below. We are
-a friendly, collaborative group and look forward to working together!
+There is a lot to do! If you are interested in getting involved, here's how you can start:
+
+1. Check out our [contributor guide](https://github.com/open-telemetry/community/tree/main/guides/contributor) to learn about our contribution process.
+2. Join our [Slack](#slack), attend our [community meetings](#calendar), and subscribe to our [mailing lists](#mailing-lists).
```

Comment:
I wonder, are the mailing lists something we actively use? cc @open-telemetry/governance-committee

Reply:
@zacharyrgonzales can you remove the mailing lists? They are heavily under-utilized and people send questions to it without ever getting an answer, so while we keep them and need to consider what to do with them in the future this is misleading right now.

Reply:
The change was made in [848693e](https://github.com/open-telemetry/community/pull/2407/commits/848693eef5113720daf142d2ae6e7cd0eb728f47).